cmd/dra-example-controller/driver.go

```diff
@@ -244,18 +246,20 @@ func (d driver) unsuitableNode(ctx context.Context, pod *corev1.Pod, allcas []*c
 
 	perKindCas := make(map[string][]*controller.ClaimAllocation)
 	for _, ca := range allcas {
-		var kind string
-		switch ca.ClaimParameters.(type) {
-		case *gpucrd.GpuClaimParametersSpec:
-			kind = gpucrd.GpuClaimParametersKind
+		switch ca.Claim.Spec.ParametersRef.Kind {
+		case gpucrd.GpuClaimParametersKind:
+			perKindCas[gpucrd.GpuClaimParametersKind] = append(perKindCas[gpucrd.GpuClaimParametersKind], ca)
+		default:
+			return fmt.Errorf("unsupported ResourceClaimParameters kind: %+v", ca.Claim.Spec.ParametersRef.Kind)
```

Comment:
Can we use `unknown` instead of `unsupported` to be consistent with line 93.

Reply:
Done.